immersion/kubeflow_pipelines/cicd/solutions/lab-03_vertex.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>nit typo: GitHub instead of Github</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>